language_service/src/state/tests.rs

```diff
@@ -116,7 +116,7 @@ async fn close_last_doc_in_project() {
         .update_document(
             "project/src/this_file.qs",
             1,
-            "/* this should not show up in the final state */ we should not see compile errors",
+            "namespace Foo.Bar { operation Main() : Unit { 5 }  }",
```

Comment:
Why did you have to change this test string? I think I'd feel better knowing that we're still getting a parse error here...